themes/socialbase/templates/block/block--social-post.html.twig

```diff
@@ -45,7 +45,14 @@
  * @ingroup templates
  */
 #}
-<div class="card card--stream">
+{%
+  set classes = [
+  'card',
+  'card--stream',
+  extra_stream_class,
```

Comment:
Not sure, because before it was added just in div manually.

![Снимок экрана 2020-10-20 в 12 09 54](https://user-images.githubusercontent.com/50984627/96565541-2dbeb280-12cd-11eb-8551-d4db02354b10.png)